slides/k8s/staticpods.md

```diff
@@ -0,0 +1,239 @@
+# Static pods
+
+- Pods are usually created indirectly, through another resource:
+
+  Deployment, Daemon Set, Job, Stateful Set ...
+
+- They can also be created directly
+
+- This can be done by writing YAML and using `kubectl apply` or `kubectl create`
+
+- Some resources (not all of them) can be created with `kubectl run`
+
+- Creating a resource with `kubectl` requires the API to be up
```

Comment:
I'd probably say "requires the API to be available" so as not to be confusing here.

Reply:
Yes! Updating.